packages/nextjs/src/config/webpack.ts

```diff
@@ -484,7 +500,6 @@ export function getWebpackPluginOptions(
     authToken: process.env.SENTRY_AUTH_TOKEN,
     configFile: hasSentryProperties ? 'sentry.properties' : undefined,
     stripPrefix: ['webpack://_N_E/'],
-    urlPrefix,
```

Comment:
I have removed this because:

(a) urlPrefix is specified via `include` and so it has no effect
(b) the correct value depends on whether it's a server or client build

I could update it to `isServer ? serverUrlPrefix : clientUrlPrefix` if we still want this.